modules/virt-cclm-reqts.adoc

```diff
@@ -0,0 +1,19 @@
+// Module is included in the following assemblies:
+//
+// * virt/live_migration/virt-about-cross-cluster-live-migration.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="virt-cclm-reqts_{context}"]
+= Cross-cluster live migration requirements
+
+Cross-cluster live migration has the following requirements:
+
+* {VirtProductName} 4.20 or later must be installed.
+* The pre-release version of the {mtv-first} must be installed.
+* The {product-title} and {VirtProductName} minor release versions must match (e.g., 4.20.z).
+* Two clusters are required, and the migration network for both clusters must be connected to the same L2 network segment.
+* You must have cluster administration permissions and appropriate RBAC privileges to manage VMs on both clusters.
+* You must configure a dedicated Multus network.
```

Comment:
Isn't this repeating what we said two lines above `Two clusters are required, and the migration network for both clusters must be connected to the same L2 network segment.`?

Reply:
@awels,  I'm not sure where this is repeated. Are you talking about the Multus mention? 

Maybe change to "Two clusters are required, and the migration network for both clusters must be connected to the same L2 Multus network segment."?

Reply:
I am saying the line `You must configure a dedicated Multus network.` seems redundant because the other line exists.